pkgs/development/compilers/ghcjs/8.10/git.json

```diff
@@ -1,6 +1,6 @@
 {
   "url": "https://github.com/obsidiansystems/ghcjs",
   "rev": "9fc935f2c3ba6c33ec62eb83afc9f52a893eb68c",
-  "sha256": "sha256:063dmir39c4i1z8ypnmq86g1x2vhqndmdpzc4hyzsy5jjqcbx6i3",
+  "sha256": "063dmir39c4i1z8ypnmq86g1x2vhqndmdpzc4hyzsy5jjqcbx6i3",
```

Comment:
I think this is a generated file, doesn't make much sense to roll back…

Reply:
If that is the case then it is reverted with the next update. Not a big deal.